plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -123,9 +127,22 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
         super(props);
         this.state = {
             dropdownOpen: false,
+            disableButtonsClosingRHS: false,
         };
     }
 
+    componentDidUpdate(prevProps: ChannelHeaderPlugProps) {
+        if (prevProps.sidebarOpen && !this.props.sidebarOpen) {
+            // eslint-disable-next-line react/no-did-update-set-state
+            this.setState({disableButtonsClosingRHS: true});
+
+            setTimeout(() => {
```

Comment:
Probably nothing to worry about, but opening and closing really fast may cause the buttons being enabled too soon. Nevertheless, since we are talking about fractions of seconds here, probably it is something we don't have to worry about.